m3-sys/llvm/llvm9bindings/src/m3makefile

```diff
@@ -59,6 +61,7 @@ if not defined("LLVM_LIB_DIR")
     LLVM_LIB_DIR = PickDir("LLVM_LIB_DIR", [
         {"AMD64_LINUX"  : "/usr/local/lib", 
          "AMD64_LINUX"  : "/usr/local/llvm-3.6.1/lib", % Rodney Bates
+            "AMD64_NT"  : "/opt/llvm/lib", % VVM, de-facto it's C:\opt\llvm\lib\ or D:\opt\llvm\lib\
```

Comment:
`it is`

Reply:
It's fixed. See https://github.com/VictorMiasnikov/cm3/commit/b846c7e57e251066558c684f5c6c282d8bd23b91